apache/zeppelin
# [ZEPPELIN-2341] Remove -Psparkr build profile

### What is this PR for?
Currently users who build Zeppelin from source need to include `-Psparkr` to use `%r` with embedded local Spark. But it's quite inconvenient to write this build profile every time we build i think. So I removed `-Psparkr` and make `r` related libraries automatically downloaded when we build Zeppelin like I did #2213 

### What type of PR is it?
Improvement

### Todos
* [x] - remove the rest of `-Psparkr` build profile in `dev/create_release.sh`, `dev/publish_release.sh`, and `docs/install/build.md` after #2213 merged

### What is the Jira issue?
[ZEPPELIN-2341](https://issues.apache.org/jira/browse/ZEPPELIN-2341)

### How should this be tested?
1. Apply this patch
2. Build source with below command
```
mvn clean package -DskipTests -pl 'zeppelin-interpreter, zeppelin-zengine, zeppelin-server, zeppelin-display, spark, spark-dependencies'

```
Aftr this step, there will be `R` dir under `ZEPPELIN_HOME/interpreter/spark`. Before this PR, only `dep` dir and `zeppelin-spark_2.10-0.8.0-SNAPSHOT.jar` is generated without `-Psparkr` build profile.

4. Restart Zeppelin. To make sure, run R tutorial note under `Zeppelin Tutorial` folder

It should be run successfully without any error

### Screenshots (if appropriate)
If we build without `-Psparkr`  
- before : R related properties are not activated by default in Spark interpreter 
![screen shot 2017-04-03 at 4 31 49 pm](https://cloud.githubusercontent.com/assets/10060731/24599560/b952e414-188b-11e7-80db-ac649c869c02.png)

 - after 
![after](https://cloud.githubusercontent.com/assets/10060731/24599567/bc513a94-188b-11e7-9e93-7abca3428279.png)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
